src/lib/types.ts

```diff
@@ -820,6 +820,8 @@ export type AuthMFAEnrollResponse =
            * to use it. Avoid loggin this value to the console. */
           uri: string
         }
+        /** Friendly name of the factor, useful for distinguishing between factors **/
+        friendly_name: string
```

Comment:
Shouldn't it be `friendly_name?: string`. It will only be there if it's been added previously, right?

Reply:
Good catch, thanks!